src/shiftedRootNormLhalfBinf.jl

```diff
@@ -52,25 +52,24 @@ function prox!(
   q::AbstractVector{R},
   σ::R,
 ) where {R <: Real, V0 <: AbstractVector{R}, V1 <: AbstractVector{R}, V2 <: AbstractVector{R}}
-  γλ = 2 * σ * ψ.λ
-  ϕ(z) = acos(γλ / 8 * (abs(z) / 3)^(-3 / 2) + 0im)
-  # p = 54^(1/3) * (γλ^(2/3)) / 4 # not necessary to compute
+  νλ = 2 * σ * ψ.λ
```

Comment:
Looks like this is only used in the next line. Could you put it in there explicitly?

Reply:
Sure.